tools/simulator/libsimulator/lib/runtime/RuntimeCCSImpl.cpp

```diff
@@ -3,7 +3,7 @@
 #include "ConfigParser.h"
 #include "cocostudio/CocoStudio.h"
 #include "ui/UIHelper.h"
-#include "tinyxml2/tinyxml2.h"
```

Comment:
Got it, but it is easy to change back to use `tinyxml2/tinyxml2.h` because we don't test the case you meet and current implementation can reduce the probability of naming conflict.